src/components/CheckboxInput/CheckboxInput.module.css

```diff
@@ -1,71 +1,64 @@
 @import '../../design-tokens/mixins.css';
 
-/*------------------------------------*\
-    # CHECKBOX INPUT
-\*------------------------------------*/
-
-/**
- * Wraps the visually hidden checkbox input element and the visible sibling svg element.
- */
-.input__wrapper {
+.wrapper {
   position: relative;
   /* Centers the checkbox icon in the wrapper. */
   display: inline-flex;
   align-items: center;
   /* Aligns the checkbox with the first line of the label. */
   align-self: flex-start;
 }
-/**
- * The visually hidden input element for the checkbox. The visual checkmark is provided by an svg element.
- */
+
 .checkbox__input {
-  cursor: pointer;
-  /* Removes default margins placed by browser for checkboxes. */
+  height: 18px;
+  width: 18px;
```

Comment:
TODO: figure out what to do with these hard coded values.

For some reason this implementation needs different height/width to get the same visual size as the previous svg implementation.

Reply:
Maybe the magic numbers are fine for now? 😓 

There already some there (the height/width on the svg element), so at the very least I didn't make things worse.

Reply:
Not a css expert or EDS pundit by a long stretch, so def ok deferring to other opinions. My gut says this is not ideal but fine. Imo this makes more sense to me than the arbitrary values that were here previously.

An alternative I think you mentioned is replacing `18px` and `3px` with eds constants. Obvi changes the sizing of things, but if EDS folks are ok with it then I'd prefer that I think

Reply:
If you really wanted, you could probably do something like 
```
.checkbox__input {
  box-sizing: content-box;
  height/width: calc(var(--eds-size-2) - var(--eds-border-width-md));
  ...
  border: var(--eds-border-width-md)) solid currentColor;
}
```

and create a tertiary token for the `3px` for checkbox use, but it gets unwieldy and I think having these values is fine for now?